docs/rules/avoid-dispatching-multiple-actions-sequentially.md

```diff
@@ -1 +1,60 @@
 # avoid-dispatching-multiple-actions-sequentially
+
+> This rule promotes disallows dispatching multiple actions one after another.
+
+## Rule Details
+
+An action should be an event which abstracts away the details of store internals.
+An action can be a composition of several events, in which case the developer might be tempted to dispatch several actions in sequence. But a better approach is t dispatch one "combining" action, which exactly describes what that event entails, and then map it to several other actions in an Effect
+
+Examples of **incorrect** code for this rule:
+
+```ts
+export class Component implement OnInit {
+  constructor(
+    private readonly store: Store<AppState>,
+  ) {}
+
+  ngOnInit() {
+    this.store.dispatch(loadEmployeeList());
+    this.store.dispatch(loadCompanyList());
+    this.store.dispatch(cleanData());
+  }
+}
+```
+
+Examples of **correct** code for this rule:
+
+```ts
+// in component code:
+export class Component implement OnInit {
+  constructor(
+    private readonly store: Store<AppState>,
+  ) {}
+
+  ngOnInit() {
+    this.store.dispatch(componentLoaded());
+  }
+}
+
+// in effect:
+export class Effects {
+
+  loadInitialDataForComponent$ = createEffect(() => this.actions.pipe(
+    ofType(componentLoaded),
+    switchMap(() => [
+      loadEmployeeList();
+      loadCompanyList();
+      cleanData();
+    ]),
+  ));
```

Comment:
Actually, doing this is also a bad-practice for the same reasons as this rule.
An Effect should also result in one action that's dispatched, instead of 3.

We recommend to listen to the same action in 3 effects. For example a `loadEmployeeList` effect that makes the http request and dispatches a succes, or a failed action back to the store

Reply:
I see, I will change the example

Reply:
Done ✔️